extensions/interactions/TextInput/TextInputPredictionServiceSpec.js

```diff
@@ -28,7 +28,37 @@ describe('Text Input Prediction Service', function() {
       predictionService = $injector.get('TextInputPredictionService');
       $rootScope = $injector.get('$rootScope');
       $scope = $rootScope.$new();
+      SVMPredictionService = $injector.get('SVMPredictionService');
+      TextInputTokenizer = $injector.get('TextInputTokenizer');
+      CountVectorizerService = $injector.get('CountVectorizerService');
     }));
+    it('should have same prediction as Oppia-ml', function() {
+      // Correctness test, makes sure oppia-ml prediction is the
+      // same as front end version.
+      jasmine.getJSONFixtures().fixturesPath = 'base/core/tests/data';
+
+      var classifierData = getJSONFixture('text_input_classifier_data.json');
+      var trainingData = getJSONFixture('text_input_training_data.json');
```

Comment:
You are not using the `text_classifier_results.json` file. Also, I am assuming that this file contains the predictions obtained from oppia-ml? You should be comparing answer stored in that file with the answer obtained from frontend.